tests/e2e/demo-homepages.spec.ts

```diff
@@ -1,13 +1,15 @@
+import { isTesting } from '@lmc-eu/spirit-common/constants/environments';
+import { SERVERS, getDevelopmentEndpointUri } from '@lmc-eu/spirit-common/constants/servers';
 import { expect, test } from '@playwright/test';
 
 test.describe('Demo Homepages', () => {
   const demos = [
     {
-      url: 'https://spirit-design-system-demo.netlify.app/',
+      url: isTesting() ? SERVERS.TESTING.web : getDevelopmentEndpointUri('web'),
       package: 'web',
     },
     {
-      url: 'https://spirit-design-system-react.netlify.app/',
+      url: isTesting() ? SERVERS.TESTING['web-react'] : getDevelopmentEndpointUri('web-react'),
```

Comment:
does this work for you? Because afaik we need to test against network URLs. Eg. 192.168.64.1

Reply:
Fixed.